### PR TITLE
AY-5418_Match `SequenceInfo.frames` return type to its type hint

### PR DESCRIPTION
**Pull request by tweak-wtf**
## Changelog Description
- refactors `SequenceInfo` to match type hints as raised here: https://github.com/ynput/LabLib/pull/35
- bugfix: returns padding from highest frame in sequence

### CODE REVIEW

File: lablib/lib/imageio.py [hunk 1]
```diff
@@ -283,33 +283,17 @@ def format_string(self) -> str:
...
-        result = f"{basename}.%0{self.padding}d{ext}"
-        return result
-
     @property
     def padding(self) -> int:
         """:obj:`int`: The sequence's frame padding."""
-        frame = min(self.frames)
-        result = len(str(frame.frame_number))
-        return result
+        return len(str(self.start_frame))
```
Comment (jakubjezek001) [inline lablib/lib/imageio.py]: Shouldn't we rather use end frame here? It usually has the higher padding value (ex. 999-1020)

Comment (jakubjezek001) [inline lablib/lib/imageio.py]:
```suggestion
        return len(str(self.end_frame))
```

Comment (tweak-wtf) [reply lablib/lib/imageio.py]: so so true!
will fix when i get home today 👌

**Review by jakubjezek001 — APPROVED** [review]
Great! good to be merged ;)

**Pull request merged**